### ISSUE TITLE
Where is binary located on Windows when installing from *.msi or with `winget`?

### ISSUE THREAD

**Issue by inliquid** [tetratelabs/wazero#1374]
I just tried `winget install wazero` on Windows 10 and realized it provides zero information regarding where the binary is located on the system.

It's not available nowhere within the `%PATH%`.

When installing with `--verbose` flag it just says that installer was taken from:
https://github.com/tetratelabs/wazero/releases/download/v1.0.1/wazero_1.0.1_windows_amd64.msi

And running it manually also gives zero knowledge of where to locate the binary. I'm not even sure that it is saved after all.

**Comment by codefromthecrypt** [comment]
Hi, sorry for your frustrations. Certainly, that's a non-goal!

This should place the wazero binary into to [program files 64 directory under wazero](https://github.com/tetratelabs/wazero/blob/main/packaging/msi/wazero.wxs#L78-L79). Also, it should be added to the PATH (which is a [part of our tests](https://github.com/tetratelabs/wazero/blob/main/packaging/msi/verify_msi.cmd#L22-L26), to ensure it is available). Note you have to refresh your shell to pickup PATH additions.

Do other winget installers introduce a manual step to control the installation folder or echo it to the screen? I'm wondering how we can do this more differently, and examples help.

**Comment by inliquid** [comment]
Thank you, so effectively `wazero` binary was installed in:
`C:\Program Files\wazero` dir. (Shame that `winget` does not provide that info, there is already an issue: https://github.com/microsoft/winget-cli/issues/2298.)

However it's not in the `%PATH%` for some reason. I double-checked and it's not there.

I would of course install just by compiling it with `go`, but wanted to try that way as well.

**Comment by codefromthecrypt** [comment]
I'll leave this open as I don't have my windows laptop with me. If someone else can take a look at if they also don't get PATH update and report back, much obliged.

**Comment by evacchi** [comment]
yep I can confirm I can't find it in neither in my User %PATH% neither in the system %PATH% :/

**Comment by codefromthecrypt** [comment]
@evacchi maybe have a look at func-e and/or podman to see if there was any drift or differences? (e.g. does winget func-e still work?) Maybe something skewed as sometimes utility upgrades break things (e.g. the thing that builds the installer).

**Comment by evacchi** [comment]
same with func-e @codefromthecrypt TBH I noticed that something wasn't registered to PATH even with `choco` while setting this machine up, so it may not one specific issue of ours 🤔 nevertheless...

**Comment by evacchi** [comment]
hey @inliquid can you try the artifact attached here? https://github.com/evacchi/wazero/actions/runs/4764921388 it's a fixed MSI built via issue #1385

**Comment by inliquid** [comment]
@evacchi yep, it worked!

**Comment by codefromthecrypt** [comment]
hopefully this works! https://github.com/microsoft/winget-pkgs/pull/103155

**Comment by evacchi** [comment]
just installed from winget, looks good to me:


```
C:\Users\ADMIN>where wazero
C:\Users\ADMIN\go\bin\wazero.exe
C:\Program Files\wazero\wazero.exe

C:\Users\ADMIN>del C:\Users\ADMIN\go\bin\wazero.exe

C:\Users\ADMIN>wazero
wazero CLI

Usage:
  wazero <command>

Commands:
  compile       Pre-compiles a WebAssembly binary
  run           Runs a WebAssembly binary
  version       Displays the version of wazero CLI

C:\Users\ADMIN>where wazero
C:\Program Files\wazero\wazero.exe
```
